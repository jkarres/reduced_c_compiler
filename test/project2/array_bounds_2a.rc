// writing to an invalid location

function : void main() {
  int[9] ia;

  int i;

  for (i = 0; i < 10; ++i) {
    ia[i] = i;
  }

}
