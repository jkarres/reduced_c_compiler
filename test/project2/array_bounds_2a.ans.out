Index value of 9 is outside legal range [0,9).
